homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -334,7 +346,8 @@ def parse_hue_api_put_light_body(request_json, entity):
 
         elif (entity.domain == "script" or
               entity.domain == "media_player" or
-              entity.domain == "fan"):
+              entity.domain == "fan" or
+              entity.domain == "cover"):
```

Comment:
This section should probably be changed to use the domain constants.
Also can be converted to `elif entity.domain in [script.DOMAIN, media_player.DOMAIN, ... ]:`

Reply:
This won't work so I kept it the way it is